smart-web-server/src/main/java/org/smartdata/server/generated/model/AuditOperationDto.java

```diff
@@ -0,0 +1,48 @@
+package org.smartdata.server.generated.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import javax.annotation.Generated;
+
+/**
+ * Operation of the audit event
+ */
+
+@Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
+public enum AuditOperationDto {
```

Comment:
I think the postfix `Dto` is redundant

Reply:
This class is generated by `openapi-generator-maven-plugin` and, unfortunately, there's no option for specifying postfix exclusively for enums (`modelNameSuffix` [option](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-maven-plugin/README.md) if for both model classes and enums).